README.md

```diff
@@ -27,6 +27,7 @@ Think you've found a bug? Check your understanding first by writing the mailing
     - [vim-pony](https://github.com/jakwings/vim-pony)
     - [pony.vim](https://github.com/dleonard0/pony-vim-syntax)
     - [currycomb: Syntastic support](https://github.com/killerswan/pony-currycomb.vim)
+    - [SpaveVim](http://spacevim.org), available as layer for Vim and [Neovim](https://neovim.io). Just follow [installation instructions](https://github.com/SpaceVim/SpaceVim) then put `call SpaceVim#layers#load('lang#pony')`inside configuration file (*$HOME/.SpaceVim.d/init.vim*)
```

Comment:
`SpaveVim` is a typo?

Reply:
Seems to be @jemc, if it looks good otherwise, we can edit inline and merge.

Reply:
@SeanTAllen - I keep forgetting that we can do that for other people's PRs! It's actually a little bit creepy, but it works...

Done.